File: 2060.js
const fs = require('fs');  
const data = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);

let idx = 0;
let n = data[idx++];
let divisores = [2, 3, 4, 5];
let cont = [0, 0, 0, 0];

for (let i = 0; i < n; i++) {
    let l = data[idx++];
    
    for (let j = 0; j < divisores.length; j++) {
        if (l % divisores[j] === 0) {
            cont[j]++;
        }
    }
}

for (let j = 0; j < divisores.length; j++) {
    console.log(`${cont[j]} Multiplo(s) de ${divisores[j]}`);
}

File: 2060.py
n = int(input())
valores = list(map(int, input().split()))
divisores = [2, 3, 4, 5]
cont = [0, 0, 0, 0]

for l in valores:
    for i in range(4):
        if l % divisores[i] == 0:
            cont[i] += 1

for i in range(4):
    print(f"{cont[i]} Multiplo(s) de {divisores[i]}")
